SDWebImageWebPCoder.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be ignored. 😊😊

Reply:
Yes, I will add xcshareddata to the .gitignore